### ISSUE TITLE
Hover and focus indicators on header nav items have insufficient contrast 

### ISSUE THREAD

**Issue by amellnik** [angular/material.angular.io#759]
![image](https://user-images.githubusercontent.com/7132414/81487003-2f5c5880-920e-11ea-9102-e37d41cfeab7.png)

In the image above, Components is hovered with #6338b0 and CDK is focused with #5B33A2.  The background is #673AB7 which makes the contrast ratios 1.05:1 and 1.17:1 respectively.  WCAG 2.1 requires a contrast ratio of at least 3:1 for UI elements.  

This is for the default theme, but the other themes have similarly low contrast ratios.

**Comment by Splaktar** [comment]
Thank you for this report!

For our own reference, here's the WCAG 2.1 link to UI "state" decoration contrast guidance: https://www.w3.org/TR/WCAG21/#non-text-contrast. It does say `3:1` contrast.
